crates/core/component/dex/src/circuit_breaker/execution.rs

```diff
@@ -0,0 +1,36 @@
+const MAX_PATH_SEARCHES: u32 = 64;
```

Comment:
Can we make this a parameter of `ExecutionCircuitBreaker`? Keeping the constant to initialize the code

Reply:
This is already the case